docs/customer_management/customer_portal_guide.md

```diff
@@ -0,0 +1,189 @@
+
+### Self registration
+
+Self-registration allows your customers to take control of their business accounts.
+This not only improves customer satisfaction but also reduces the administrative burden on your team. 
+With the ability to integrate with [[= product_name_connect =]] you are able to fully automate the process.
+
+### Address book
+
+Use of an address book allows you to add many shipping addresses to one company for clients with multiple locations.
+
+### Custom prices
+
+You can offer special prices and additional discounts dedicated for Customer groups containing company members with verified accounts.
+
+### Available in segments
+
+Corporate accounts are available in segments, which means you can assign companies to different recommendation groups based on gathered data, and deliver recommendations.
+It allows you to make use of customer targeting of the segments and create personalized experience for each company.
+
+## Benefits
+
+### General overview
+
+The overall benefit of customer portals is the help they provide to retain customers and increase loyalty, while freeing up customer service employees time for higher-level work.
+They can achieve that by providing customers with up-to-date information about their orders and deliveries, personalize shopping experience, offer special deals available only to B2B partners and do that in one, easily accessible space.
+Currently, customer portals are a standard in global sites such as Amazon. They are the level of quality that customers just expect, and all businesses strive for.
+
+### Simplified shopping process
+
+Business account helps streamline the B2B shopping process with all the paperwork, payment, 
+and other administrative tasks converted into a few simple steps with prefilled forms, billing addresses, shipping addresses and more. 
+Making your site a go-to place for company orders.
+
+### Better customer experience
+
+In the era of internet, customers expect quick, accessible and excellent quality service and user experience from every business they associate with. 
+Customer portals offer a seamless self-service experience by providing complete 24/7 access to relevant, up-to-date information and customer support.
+
+### Client encouragement
+
+Price strategies are a great way to build and maintain strong relationships with your trading partners. 
+With special prices available to B2B clients, you can offer the best deals in highly competitive markets. 
+Those discounts may be a great encouragement to convince big buyers to choose your business over other options. 
+Competitive prices impact not only the size of the customer base, 
+they affect every customer’s purchasing strategy, including the diversity, frequency, and volume of their orders.
+
+### Cost benefits
+
+Customer portals help you to automate tasks that otherwise would be done by your employees manually, such as customer services, checking shipment status. 
```

Comment:
```suggestion
Customer portals help yo automate tasks that otherwise would be done by your employees manually, such as customer services, or checking shipment status. 
```